meetings/160120_organisers.md

```diff
@@ -24,3 +25,169 @@ Agenda:
+
+
+###Workshops in 2015###
+
+Hugues was not present and could not summarized the 2015 Software Carpentry Surveys. However, Lex & Karin gave a short summary:
+
+    - UIO held 3 workshops in 2015. 
+    - All of them were a combination of shell, git and python. 
+    - 101 persons registered to these workshops and 70 persons attended.
+
+
+###Email lists###
+
+Several Email lists were opened and are now fully operational:
+
+- announcements@swcarpentry.uio.no
+- discussions@swcarpentry.uio.no
+- organizers@swcarpentry.uio.no
+
+Ben mentioned he sent an email to this list but nobody received it. Karin replied that he sent it before the lists were up and running.
+
+###Actions###
+- Lex, Karin and Arnstein will continue their effort to find a suitable host at UIO to become member of the Software Carpentry Foundation and they will inform us as soon as a solution is found.
+- Hugues will send the survey summary to Lex (to be incorporated in the report in preparation).
+- Ben will resent the email (podcast on Software & Data Carpentry goals) to the list
+
+## Workshops in 2016##
+We plan to organize about 7 Software Carpentry workshops in 2016 as well as a Carpentry Week (March 2016)
+
+    - February
+    - March (Carpentry week, see own item)
+    - April
+    - May/June
+    - September
+    - October
+    - November
+    - early December
+
+###General organization of Software Carpentry Workshop###
+The organization procedures were discussed and in general were found satisfying. As last year, we agreed to give priority to those who are in the waiting list for the registration. 
+
+Ben said that we should encourage attendees to come with colleagues as recommended by Software Carpentry. We decided to follow this recommendation and discussed on how to implement it:
+- add an item in the registration form to name possible colleagues wishing to attend the workshop
+- make sure we respect these wishes (especially when the workshop is full/nearly full and we have to pick up a few persons only).
+
+Ben mentionned recurring setup problems with attendees: several did not follow the setup instructions and did not install the necessary tools. However, we only discovered it on the second day when using git. We decided to check the availability of all the necessary tools (python, git, etc.) at the beginning of a Workshop to avoid wasting time on the second day.
+
+
+Lex also pointed an issue with the room booking system at the Science Library. It does not allow to book a room more than a month in advance which delays advertisement of Software Carpentry workshops and as a consequence the registration process. No specific action was taken to circumvent this problem. 
```

Comment:
Well, what I meant was that we are allowed to use the Bikuben area at the biology buidling, but cannot ask for a reservation more than a month in advance. And in general, geeting a room for two full days is difficiult at UiO